allowed_planning_time: 20.0
pose_goal_planner_id: right_arm_and_torso[right_arm_and_torso_ARA_BFS_ML]
joint_goal_planner_id: right_arm_and_torso[right_arm_and_torso_ARA_JD_ML]
group_name: right_arm_and_torso
pos_tolerance: 0.01
rot_tolerance: 5.0
joint_tolerance: 5.0
tip_link: limb_right_link7
workspace_frame: base_footprint

workspace_min:
    x: -0.5
    y: -1.5
    z: 0.0
workspace_max:
    x: 1.5
    y: 0.5
    z: 1.9
